### DISCUSSION TITLE
Developer Mode Menu Checkbox - What does it do?

### DISCUSSION THREAD

**Discussion by maxoakland** [LIJI32/SameBoy#647] [Q&A] [answered]
I've looked through the interface to see if anything changes depending on if Developer Mode is Checked or not, but I can't figure it out. Does anyone know?

**Comment by LIJI32** [comment, marked as answer]
Developer mode enables the debugger. It enables breaking the debugger with ^C, as well as allows the console to automatically open when a ROM issues a warning.

**Reply by maxoakland** [reply]
Thanks for explaining!
